hhvm.rb

```diff
@@ -241,6 +241,14 @@ def install
       -DZLIB_LIBRARY=/usr/lib/libz.dylib
     ]
 
+    # Compile with PGSQL
+    cmake_args += %W[
+      -DENABLE_EXTENSION_PGSQL=ON
+      -DPGSQL_INCLUDE_DIR=/usr/local/opt/postgres/include
+      -DPGSQL_LIBRARY=/usr/local/opt/postgres/lib/libpq.dylib
```

Comment:
Sorry, I didn't quite think this through, haha.  Orvid just told me to make a pull request with the updated homebrew recipe.